File: CS/ControlTemplateGallerySample/ControlTemplateGallery/TemplateEditorForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Linq;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace ControlTemplateGallerySample
{
    public partial class TemplateEditorForm : DevExpress.XtraEditors.XtraForm
    {
        public TemplateEditorForm()
        {
            InitializeComponent();
        }

        public string CategoryName 
        {
            get { return teCategoryName.Text; }
            set { teCategoryName.Text = value; }
        }

        public string TemplateName
        {
            get { return teTemplateName.Text; }
            set { teTemplateName.Text = value; }
        }
    }
}

File: VB/ControlTemplateGallerySample/ControlTemplateGallery/TemplateEditorForm.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace ControlTemplateGallerySample
	Partial Public Class TemplateEditorForm
		Inherits DevExpress.XtraEditors.XtraForm

		Public Sub New()
			InitializeComponent()
		End Sub

		Public Property CategoryName() As String
			Get
				Return teCategoryName.Text
			End Get
			Set(ByVal value As String)
				teCategoryName.Text = value
			End Set
		End Property

		Public Property TemplateName() As String
			Get
				Return teTemplateName.Text
			End Get
			Set(ByVal value As String)
				teTemplateName.Text = value
			End Set
		End Property
	End Class
End Namespace